pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -78,5 +81,32 @@ func (m *kubeGenericRuntimeManager) generateLinuxContainerConfig(container *v1.C
 		lc.Resources.CpuPeriod = cpuPeriod
 	}
 
+	pageSizes := sets.NewString()
+	for resourceObj, amountObj := range container.Resources.Limits {
+		if v1helper.IsHugePageResourceName(resourceObj) {
+			pageSize, err := v1helper.HugePageSizeFromResourceName(resourceObj)
+			if err != nil {
+				continue
```

Comment:
Should we issue a warning instead of silently ignoring this error?

Reply:
Adding warning message looks more reasonable. I will add it.

Reply:
updated.